Packs/XMCO/Integrations/Serenety/Serenety.yml

```diff
@@ -0,0 +1,72 @@
+fromversion: 6.10.0
+category: Vulnerability Management
+sectionOrder:
+  - Connect
+  - Collect
+commonfields:
+  id: Serenety
+  version: -1
+configuration:
+- display: XMCO LePortail instance URL
+  defaultvalue: https://leportail.xmco.fr/
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- displaypassword: API Key
+  name: api_key
+  required: true
+  type: 9
+  hiddenusername: true
+  section: Connect
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  section: Collect
+- display: Maximum number of incidents per fetch
+  defaultvalue: '50'
+  name: max_fetch
+  type: 0
+  required: false
+  section: Collect
+- defaultvalue: 3 days
+  display: First fetch time
+  additionalinfo: First alert created date to fetch.
+  name: first_fetch
+  type: 0
+  required: false
+  section: Collect
+- display: Incident type
+  name: incidentType
+  type: 13
+description: 'Fetch Serenety alert from XMCO.'
+display: Serenety
+name: Serenety
+script:
+  commands:
```

Comment:
What about additional commands on updating the incident status on Serenety console?
OR maybe retrieve remediation steps?

Reply:
For our first client, who is in a hurry, we have deal that the first version will be simple, only retrieve the incident. The remediation steps are included with all the information in the description field. Maybe we need to improve that.

But i've already mutlitple idea for improving this integration in the next versions

Reply:
NP sounds good - resolving this comment.